test/integration/features/addon-xpaas.feature

```diff
@@ -0,0 +1,34 @@
+@addon-xpaas @addon
+Feature: XpaaS add-on
+
+  Scenario: User enables the xpaas add-on
+     When executing "minishift addons enable xpaas" succeeds
+     Then stdout should contain "Add-on 'xpaas' enabled"
+
+  Scenario: User starts Minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start" succeeds
+     Then Minishift should have state "Running"
+      And stdout should contain
+      """
+      XPaaS OpenShift imagestream and templates installed
+      """
+
+  Scenario Outline: User deploys, checks out and deletes several templates from XpaaS imagestream
+   Given Minishift has state "Running"
+    When executing "oc new-app <template-name>" succeeds
+     And services "<service-name>" rollout successfully
+    Then body of HTTP request to "<http-endpoint>" of service "<service-name>" in namespace "myproject" contains "<expected-hello>"
+     And status code of HTTP request to "<http-endpoint>" of service "<service-name>" in namespace "myproject" is equal to "200"
+     And executing "oc delete all --all" succeeds
+
+  Examples: Values to be used on next Minishift start
```

Comment:
I guess the example description is not quite right

Reply:
thanks for a catch, missed it, fixing it now